dashboard/src/App.js

```diff
@@ -1,26 +1,65 @@
-import React from 'react';
+import React, { useState } from 'react';
 import logo from './logo.svg';
 import './App.css';
 
-//component imports
-import Index from "../src/components/Index";
+/*
+class App extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      items = []
+    }
+  }
+
+  render() {
+    const { items } = this.state;
+    return <>
+      <input> </input>
+      <h1> gg </h1>
+      <ul>
+        {items.map(item => <li>{item} </li>)}
+      </ul>
+    </>
+  }
+}*/
```

Comment:
it's fine to delete code you're not using anymore instead of commenting it out, as git preserves the state of all code at each commit so you can always revert/get it back